values.schema.json

```diff
@@ -70,6 +70,12 @@
         "limits": { "type": "object", "title": "resource limits", "default": {} },
         "requests": { "type": "object", "title": "resource requests", "default": {} }
       }
+    },
+    "tls": {
+      "type": ["null", "object"], "title": "use TLS secret", "default": null,
+      "properties": {
+        "secret": {"type": ["null", "string"], "title": "Secret name to attach to the ingress object", "default": null }
```

Comment:
There is no need to make `secret` nullable. Since its optiinal propery - it could just b omitted.

Reply:
removed and left it as only string...

Reply:
Thnx